src/app/configurations/allocations/calculations/calculations.states.js

```diff
@@ -21,13 +21,8 @@ angular.module('allocations')
                 return []
               })
           },
-          locationStates: function (locationService) {
-            var keys = []
-            keys.push(['2', 'NG']) // TODO: get this from auth
-            return locationService.getByLevelAndAncestor(keys)
-              .catch(function () {
-                return []
-              })
+          locationStates: function (locationService, userStateService) {
+            return userStateService.getUserSelectedState('object')
```

Comment:
OK, but maybe it'd be better to split that method into several helpers with descriptive names rather than relying on magic strings/params.